daprdocs/content/en/reference/components-reference/supported-middleware/middleware-bearer.md

```diff
@@ -48,19 +54,7 @@ Common values for `issuer` include:
 
 ## Dapr configuration
 
-To be applied, the middleware must be referenced in [configuration]({{< ref configuration-concept.md >}}). See [middleware pipelines]({{< ref "middleware.md">}}).
```

Comment:
You should keep this change but use this to have a consistent warning after Dapr configuration title to say that this is no longer recommended.